frontend/android/src/main/java/com/jakewharton/sdksearch/ui/MainActivity.kt

```diff
@@ -154,6 +159,22 @@ class MainActivity : Activity() {
           queryInput.typeface = if (empty) Typeface.DEFAULT else robotoMono
         }
 
+    var totalDy = 0
+    val vc = ViewConfiguration.get(recycler.context)
+    val slop = vc.scaledTouchSlop
+    recycler.scrollEvents()
+            .filter({ scroll -> scroll.dy() > 0 })
+            .observeOn(mainThread())
+            .crashingSubscribe {
+              scroll -> totalDy += scroll.dy()
+              if (totalDy >= slop) {
+                systemService<InputMethodManager>().apply {
```

Comment:
remove `.apply`, it's useless. just call the method on the returned value directly